docs/GITEA_GUIDE.md

```
@@ -1,4 +1,4 @@
-<!-- markdownlint-disable MD041 -->{% raw %}
+{% raw %}
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)